tests/framework/installer/install_helper.go

```diff
@@ -226,94 +226,24 @@ func (h *InstallHelper) UninstallRookFromK8s() {
 	if err != nil {
 		panic(err)
 	}
-	rookOperator := h.installData.getRookOperator(k8sVersion)
```

Comment:
for future reference, it would have been nice to have the refactoring in a separate PR from the change to test other namespaces, or at least in a separate commit in this PR.